betty/wiki/client.py

```diff
@@ -186,9 +197,24 @@ async def get_image(self, page_language: str, page_name: str) -> Image | None:
                         f"Could not successfully parse the JSON content returned by {url}: {error}"
                     )
                 ) from error
+
+            # Fetch copyright information
+            copyright = None
```

Comment:
If you do this in the `except` clause, it's clearer why this is set to `None`. You'll probably have to declare the variable's full type beforehand either way.